docs/cudf/source/user_guide/performance_comparisons.ipynb

```diff
@@ -0,0 +1,1517 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    pandas_udf_groupby, cudf_udf_groupby = bench(pdf, gdf, lambda df: df.groupby(['key']).apply(custom_formula_udf), number=1)
</pre><p>Please fix the warning in this cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/12595/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
Fixed